connection.go

```diff
@@ -85,13 +85,18 @@ func (sc *snowflakeConn) exec(
 	var err error
 	counter := atomic.AddUint64(&sc.SequenceCounter, 1) // query sequence counter
 
+	queryContext, err := buildQueryContext(sc.queryContextCache)
+	if err != nil {
+		logger.Errorf("error while building query context: %v", err)
```

Comment:
Do we continue even if error occurred?

Reply:
Yes. This is confirmed with Lianke, that we should proceed and just not send query context.